src/test/resources/good/ManyDispatchMethod.java

```diff
@@ -0,0 +1,31 @@
+package test;
+
+import com.bnorm.auto.dispatch.AutoDispatch;
+
+public class ManyDispatchMethod {
+
+  @DescriptionMulti(Age.Young)
+  public String descriptionYoung(Person person) {
+    return "You're so young!";
+  }
+
+  @DescriptionMulti({Age.Old})
+  public String descriptionOld(Person person) {
+    return "You're quite old...";
+  }
+
+  @DescriptionMulti({Age.Young, Age.Old, Age.VeryOld})
+  public String descriptionDefault(Person person) {
```

Comment:
I feel specifying nothing isn't very clear.

Another annotation would work. Though I like the uniformity of how it is currently. Adding an annotation, we would need a case to look for the annotation. Also starts getting a bit distracting when you have annotations piling up on each other.

 If you wanted to have a default in an actual project you would likely define a static Age[] DEFAULT = all enum values.